test/support/publishing_api_test_helpers.rb

```diff
@@ -47,4 +47,19 @@ def expect_no_republishing(*editions)
         .with(content_id: edition.content_id).never
     end
   end
+
+  def disable_publishes_to_publishing_api
+    all_classes = ObjectSpace.each_object(Class).select { |c| c.included_modules.include? PublishesToPublishingApi }
+    all_classes.each do |klass|
+      klass.any_instance.stubs(:publish_to_publishing_api)
+      klass.any_instance.stubs(:publish_gone_to_publishing_api)
+    end
+
+    yield
+
+    all_classes.each do |klass|
+      klass.any_instance.unstub(:publish_to_publishing_api)
+      klass.any_instance.unstub(:publish_gone_to_publishing_api)
+    end
+  end
```

Comment:
I think given the alternatives, this is a sensible approach.

If we did want to change the helpers to support `times` or change Whitehall to use queueing, we'd easily be able to find all calls to `disable_publishes_to_publishing_api`.

We should make sure we have at least some tests that use that module.